toto/util.py

```diff
@@ -97,6 +97,15 @@ def import_rsa_key_from_file(filepath, password=None):
 
   return rsa_key
 
+def import_rsa_public_keys_from_files_as_dict(filepaths):
+  """Takes a list of filepaths to RSA public keys and returns them as a
+  dictionary conformant with ssl_crypto.formats.KEYDICT_SCHEMA."""
+  key_dict = {}
+  for filepath in filepaths:
+    key = import_rsa_key_from_file(filepath)
+    keyid = key['keyid']
+    key_dict[keyid] = key
+  return key_dict
```

Comment:
It may be a good idea to add statements to check whether the arguments of this function are sane (e.g., am I even getting a list?!)